js/views/modals/About/Donations.js

```diff
@@ -9,9 +14,37 @@ export default class extends baseVw {
     });
   }
 
+  events() {
+    return {
+      'click #copyAddress': 'copyDonationAddress',
+      'click #donationAddress': 'copyDonationAddress',
+    };
+  }
+
+  copyDonationAddress() {
+    const address = document.querySelector('#donationAddress');
+    const range = document.createRange();
+    const selection = window.getSelection();
+    range.selectNodeContents(address);
+    selection.removeAllRanges();
+    selection.addRange(range);
```

Comment:
Forcing any selection to select the whole address is probably a little bit of overkill, but it's ok.